src/editor/editorconfig.jsdoc

```diff
@@ -162,6 +164,11 @@
  * * **`toolbar.viewportTopOffset`** &ndash; The offset (in pixels) from the top of the viewport used when positioning a sticky toolbar.
  * Useful when a page with which the editor is being integrated has some other sticky or fixed elements
  * (e.g. the top menu). Thanks to setting the toolbar offset the toolbar will not be positioned underneath or above the page's UI.
+ * * **`toolbar.shouldGroupWhenFull`** &ndash; When `true` (default), the toolbar will group its items that
```

Comment:
This documentation entry is growing (`items`, `viewportTopOffset` and now `shouldGroupWhenFull`). I know it's convenient for developers to keep it in a single place (`module:core/editor/editorconfig~EditorConfig#toolbar`) but if we get more options like this, I think it's time to define some `@interface ToolbarConfig` and put the detailed documentation there.

Reply:
As long as those options are fairly simple (boolean, number, simple array), I'd keep them locally here because, as you mentioned, that more convenient to read. But I agree that at some point we may need to extract it. But then we can keep a lot of information here locally, and have details in the interface.